src/genericComponents/Datepicker.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { useState } from 'react';
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+
+const CAGDatePicker = (props: any) => {
+  const [selectedDate, setSelectedDate] = useState(new Date());
+
+  return (
+    <DatePicker
+      onChange={() => setSelectedDate(selectedDate)}
+      onSelect={date => setSelectedDate(date)}
+      selected={selectedDate}
+    />
+  );
+};
+
+export default CAGDatePicker;
```

Comment:
Actually, since this component doesn't have any specific styling you're giving it, there's no reason to have a separate component for it for now. Let's see if we need any specific customizations once we start implementing the datepicker and we can revisit this component if necessary. For now, you can remove this component altogether.